src/sonic-py-common/sonic_py_common/daemon_base.py

```diff
@@ -48,6 +65,20 @@ def __init__(self, log_identifier):
         if not signal.getsignal(signal.SIGTERM):
             signal.signal(signal.SIGTERM, self.signal_handler)
 
+    def log(self, priority, msg, also_print_to_console=False):
```

Comment:
@xincunli-sonic why do we need this? Where it is being used?

Reply:
Current daemon_base is being used by most of PMON daemon process, and it is logger's subclass, not necessary to change them to inherit from syslogger, so this log function is going to be compatible between SysLogger and Logger.